content/news/release-20161019.md

```diff
@@ -0,0 +1,65 @@
+<div style="margin:0 50px; font-size: 0.9em; border-radius: 4px; background-color: #272d2c; padding: 10px">
+<pre style="margin: 0">
+LANDCOM 16 HQS.
+TOP SECRET
+TO: ALL FIELD COMMANDERS
+
+REDIRECT IMMEDIATELY TO RELEASE 20161019.
+
+A SURPRISE AMBUSH OF MULTIPLAYER ISSUES HAS LEFT THE 20161015 RELEASE
+CRITICALLY WEAKENED.  REPORTS INDICATE THE PRESENCE OF A MULTI-PLAYER
+DESYNC CRASH IN THE RED ALERT BATTLEGROUND FOLLOWING ENEMY DEPLOYMENT
+OF GPS SATELLITES.  SCRAMBLED LOBBY GAME SPEED COMMANDS ARE CONNECTED
+TO THE REDUCED FIELD MOBILITY OF SOME COMMANDERS.
+
+PROCEEED TO <a href="/download/">www.openra.net/download/</a> TO OBTAIN THE FORTIFIED RELEASE.
+RENDEZVOUS WITH OTHER COMMANDERS IN THE MULTIPLAYER LOBBY AND DESTROY
+ALL ENEMY OPPOSITION.
+
+TRANSMISSION ENDS.
+</pre>
+</div>
+
+A release-20161019 hotfix is now available to fix two critical multi-player bugs that were missed during the playtesting phase. The original release-20161015 release announcement is included below:
```

Comment:
"The original announcement for release-20161015 is included below" flows a bit better, I think.